Add documentation section and update syna (#30)
eclipse-cdt-cloud/website: content/documentation/overview/content.md

+++
fragment = "content"
weight = 10

> title = "Overvew"

Reviewer: ```suggestion
title = "Overview"
```

* wizards for creating projects and and custom editors for specific configuration artifacts
* example projects to get users up and running quickly
* excellent language editing support for C/C++ and other languages without manual user configuration
* easy-to-use debug configurations or other analysis capabilities, such as memory inspectors or trace viewers,
* and much more.

### How does CDT.cloud help?

Custom C/C++ development tools have several reoccuring capabilities, such as creating projects, editing source code, building projects, debugging and tracing applications, etc.
While they may differ from one specific tool use case to another, the base components can still be reused and adapted to the specifics of an environment.
This makes building such tools significantly more efficient.
And that's precisely where CDT.cloud is here to help: it provides a feature-rich ecosystem of reusable platforms and components that can be customized for your environment and bundled into a ready-to-use tool product for your use case.

### How do I get started?

> In most of the cases, a great starting point is to selecting a [development tool platform]({{< relref  "tool-platform" >}}) for your custom tool, which you augment with VS Code extensions too add required tool capabilities and then put customizations for your specific environment on top.

Reviewer: ```suggestion
In most cases, a great starting point is to select a [development tool platform]({{< relref  "tool-platform" >}}) for your custom tool, which you augment with VS Code extensions too add required tool capabilities and then put customizations for your specific environment on top.
```

> In most of the cases, a great starting point is to selecting a [development tool platform]({{< relref  "tool-platform" >}}) for your custom tool, which you augment with VS Code extensions too add required tool capabilities and then put customizations for your specific environment on top.

Reviewer: When I read this as a novice I don't know what is meant with "development tool platform". Clicking on the link also only refers to Theia, so I still don't know what any alternative could be. Maybe just remove this sentence completely and just start with the Blueprint?

> CDT.cloud also provides an example tool named [CDT.cloud blueprint]({{< relref  "blueprint" >}}), which may act as a template and a source of inspiration for your specific tool.

Reviewer: ```suggestion
CDT.cloud also provides an example tool named [CDT.cloud Blueprint]({{< relref  "blueprint" >}}), which may act as a template and a source of inspiration for your specific tool.
```
